### PR TITLE
Add Quick Start Guide for PingCAP Clinic

### PR DESCRIPTION
**Pull request by en-jin19**
### What is changed, added or deleted? (Required)
Add a new doc "Quick Start Guide for PingCAP Clinic"

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/8733
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: clinic/quick-start-with-clinic.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+        > **Note:**
+        >
+        > When uploading data using this method, you need to use Diag v0.7.0 or later versions. When running Diag, Diag provides its version information. If you have installed Diag in an earlier version of 0.7.0, you can use the `tiup update diag` command to upgrade Diag to the latest version.
+
+    - If the network where your cluster is located cannot access the internet, you need to pack the data and upload the folder with collected data. For details, see [Method 2. Pack and upload data](/clinic/clinic-user-guide-for-tiup.md#method-2-pack-and-upload-data).
+
+3. After the upload is complete, get the data access link from `Download URL` in the output of the command in the second step.
+
+    By default, the diagnostic data includes the cluster name, cluster topology information, log content in the collected diagnostic data, and Grafana Dashboard information reorganized based on the metrics in the collected data.
```
Comment (qiancai) [inline clinic/quick-start-with-clinic.md:105]: “the diagnostic data includes” 和 “log content in the collected diagnostic data” 看起来可能有些重复

File: clinic/quick-start-with-clinic.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+
+    You can use this data to troubleshoot cluster problems, or you can provide the link to PingCAP technical support staff to facilitate the remote troubleshooting.
```
Comment (qiancai) [inline clinic/quick-start-with-clinic.md:107]:
```suggestion
    You can use the data to troubleshoot cluster problems by yourself, or you can provide the data access link to PingCAP technical support staff to facilitate the remote troubleshooting.
```

**Review by qiancai — COMMENTED** [review]
@en-jin19 I've completed the review and added some comments. PTAL. Thanks.

File: clinic/quick-start-with-clinic.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+1. Install Diag on your control machine with TiUP using the following command:
+
+    {{< copyable "shell-regular" >}}
+
+    ```bash
+    tiup install diag
+    ```
+
+2. Log in to [Clinic Server](https://clinic.pingcap.com.cn) and select **Sign in with AskTUG** to enter the AskTUG community login page. If you do not have an AskTUG account, you can register on the login page.
```
[the remark this answers is not in the document]
Comment (en-jin19) [reply clinic/quick-start-with-clinic.md:33]: But the other way  (Sign in with Google) is only for PingCAP employees, so I guess users need to use a translation application by themselves. Or is there other option for international users? @qqqdan

File: clinic/quick-start-with-clinic.md (hunk 1 above)
Comment (en-jin19) [inline clinic/quick-start-with-clinic.md:105]:
```suggestion
    By default, the diagnostic data includes the cluster name, cluster topology information, logs, and Grafana Dashboard information reorganized based on the metrics in the collected data.
```

Comment (en-jin19) [reply clinic/quick-start-with-clinic.md:105]: @qiancai @qqqdan PTAL, thanks!

File: clinic/quick-start-with-clinic.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (qqqdan) [reply clinic/quick-start-with-clinic.md:33]: We will provide US server for international user , https://clinic.pingcap.com.cn is only for China users.

**Review by qqqdan — APPROVED** [review]

**Comment by Oreoxmt** [comment]
Close by https://github.com/pingcap/docs/pull/8958

**Pull request closed without merging**